docs/UserGuide.md

```diff
@@ -41,146 +38,415 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task with a start date to the task manager
 
-> Persons can have any number of tags (including 0)
+Format: `add DESCRIPTION from START_DATE`
+
+> Tasks with a start date will be added to the task manager.
+
+Examples:
+
+* `add write essay from 13/7/17`
+
+### 2.6. Adding a task with a duration: `add ... from ... to ...`
+
+Adds a task to the task manager
+
+Format: `add DESCRIPTION from START_DATE to END_DATE`
+
+> Tasks will be added to the task manager with a duration.
+
+Examples:
+
+* `add write essay from 13/7/17 to 15/7/17`
+
+### 2.7. Adding a task with tags: `add ... #TAGNAME ...`
+
+Adds tags to the task
+
+Format: `add DESCRIPTION #TAGNAME`
+
+> Tasks will be added to the task manager with a tags.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add write essay from 13/7/17 to 15/7/17 #school #homework`
+* `add write essay from 13/7/17 #school #homework`
+* `add write essay by 15/7/17 #blogging`
+* `add write essay #blogging`
+
 
-### 2.3. Listing all persons : `list`
+### 2.8. Listing all tasks : `list`
+
+Shows a list of all tasks in the task manager.
 
-Shows a list of all persons in the address book.<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
+### 2.9. Listing undone tasks : `list undone`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Shows a list of all undone tasks in the task manager.
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+Format: `list`
+
+### 2.10. Listing done tasks : `list done`
+
+Shows a list of all done tasks in the task manager.
+
+Format: `list`
+
+### 2.11. Edit task : `edit INDEX`
+> * Edits the task at the specified `INDEX`.
+    The index refers to the index number shown in the last tasks listing.
     The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
+> * At least one of the optional fields of the task must be provided.
 > * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+> * When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
+> * You can remove all the task's tags by editing the tag to an empty value.
+
+Examples:
+
+* `edit 1 buy groceries by 13/5/17`
+  Edits the description of the task to `buy groceries` and due date to `13/5/17` respectively.
+
+* `edit 2 buy groceries`
+  Edits the description of the 2nd task to be `buy groceries`.
```

Comment:
👍  This is good... the more flexible it is, the better.  Of course, you will need to put in a lot of work in the parser.  I assure you that if you see to it, then the payoff will be very high. 😄